src/components/Sidebar/Sidebar.tsx

```diff
@@ -65,7 +76,7 @@ function SidebarComponent({
       attach: attach === 'left' ? 'right' : 'left',
       onClick: toggle(child),
       key,
-      isActive: activeStyles && active === child.props.label,
```

Comment:
I removed `activeStyles` here because it seemed weird to derive the active state from whether `activeStyles` prop was provided. IMO it's active state is separate. The change [here](https://github.com/vimeo/iris/pull/271/files#diff-41f48a2ccb4296a2dd74286e47d6d9d9a60b1bbfbf32768ae012b4afcfcc60ffR59) accounts for this